docs/structured-streaming-programming-guide.md

```diff
@@ -1845,9 +1845,7 @@ Some of them are as follows.
   for more details.
 
 - Chaining multiple stateful operations on streaming Datasets is not supported with Update and Complete mode.
-  - In addition, below operations followed by other stateful operation is not supported in Append mode.
-    - stream-stream time interval join (inner/outer)
-    - flatMapGroupsWithState
+  - In addition, (flat)MapGroupsWithState operation followed by other stateful operation is not supported in Append mode.
```

Comment:
This includes mapGroupsWithState too right ? should we just call them out separately ?

Reply:
I thought () was sufficient - will do.